src/components/RecipeCard.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  Card,
+  CardHeader,
+  CardMedia,
+  CardContent,
+  Typography,
+  Avatar,
+  colors
+} from "@mui/material";
+
+const RecipeCard = ({ recipe }) => {
+  return (
+    <Card sx={{ maxWidth: 345, boxShadow: 3 }}>
+      <CardHeader
+        avatar={
+          <Avatar sx={{ bgcolor: colors.red[500] }}>
+            R
+          </Avatar>
+        }
+        title={recipe.title}
+        subheader="September 14, 2016"
```

Comment:
We have available date when the recipe was created, why not use it instead of hardcoded date?